library/Director/Job/SyncJob.php

```diff
@@ -103,7 +103,7 @@ public static function addSettingsFormFields(QuickForm $form)
             )
         ));
 
-        if (! strlen($form->getSentOrObjectValue('job_name'))) {
+        if (! strlen($form->getSentOrObjectValue('job_name') ?? '')) {
```

Comment:
Thanks for this PR, coalesce operator is only allowed since PHP 7.0. Requirements will be raised, but as of this writing, Director still supports 5.6